### PR TITLE
[ENTESB-14875] Avoid manual server pod respin

### PR DESCRIPTION
**Pull request by jsafarik**
//test: `@ENTESB-14875`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: operator-tests/src/test/java/io/syndesis/qe/OperatorValidationSteps.java [hunk 1]
```diff
@@ -162,6 +143,31 @@ public void deployCrFromFile(String file) {
         }
     }
 
+    private String getCrFromFileAsString(String file) throws IOException {
```
Comment (avano) [inline operator-tests/src/test/java/io/syndesis/qe/OperatorValidationSteps.java]: rather `fail` than `throws`

File: operator-tests/src/test/java/io/syndesis/qe/OperatorValidationSteps.java [hunk 2]
```diff
@@ -672,4 +678,15 @@ public void checkRepositories(String build, DataTable expectedRepositories) {
         asserts.assertThat(downloads).as("The artifact shouldn't be downloaded from other repositories than defined").isEmpty();
         asserts.assertAll();
     }
+
```
Comment (avano) [inline operator-tests/src/test/java/io/syndesis/qe/OperatorValidationSteps.java:685]: remove one empty line

**Review by avano — DISMISSED** [review]

**Review by avano — APPROVED** [review]

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert 
```

**Pull request merged**